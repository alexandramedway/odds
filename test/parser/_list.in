do []
do [1]
do [1, 2, 3]
do [42.0, 42.5]
do ["hello", "world"]
do [true, false, true]
