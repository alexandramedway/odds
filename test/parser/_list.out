[
    Do(LDecl([])) ;
    Do(LDecl([Num_lit(1)])) ;
    Do(LDecl([Num_lit(1) ; Num_lit(2) ; Num_lit(3)])) ;
    Do(LDecl([Num_lit(42.) ; Num_lit(42.5)])) ;
    Do(LDecl([String_lit(hello) ; String_lit(world)])) ;
    Do(LDecl([Bool_lit(true) ; Bool_lit(false) ; Bool_lit(true)]))
]
